app/controllers/connectiontoproperty/ConnectionToPropertySubmissionController.scala

```diff
@@ -61,6 +61,8 @@ class ConnectionToPropertySubmissionController @Inject() (
     val session                    = request.sessionData
     submitToBackend(session).flatMap { _ =>
       audit.sendExplicitAudit(auditType, submissionJson ++ Audit.languageJson)
+      // Temporary to generate failed audit event TODO REMOVE *****
```

Comment:
This audit event should be fixed. I've seen it working locally with the fix from a commit before this PR.